### DISCUSSION TITLE
Vaultwarden with Yubikey OTP

### DISCUSSION THREAD

**Discussion by amoorefilms** [tteck/Proxmox#1803] [Help Needed]
Just set up a vaultwarden lxc using the script. How do I add Yubikey support? I get this error when I try and set it up. 
![Screenshot 2023-09-16 181103](https://github.com/tteck/Proxmox/assets/23732988/b7b6739e-3d74-4353-8d47-3566d22a9e60)

**Comment by tteck** [comment]
Have you tried adding:
```
## Yubico (Yubikey) Settings
## Set your Client ID and Secret Key for Yubikey OTP
## You can generate it here: https://upgrade.yubico.com/getapikey/
## You can optionally specify a custom OTP server
YUBICO_CLIENT_ID=11111
YUBICO_SECRET_KEY=AAAAAAAAAAAAAAAAAAAAAAAA
# YUBICO_SERVER=http://yourdomain.com/wsapi/2.0/verify
```
to your `.env` file located at `/opt/vaultwarden/.env`?

https://github.com/dani-garcia/vaultwarden/blob/main/.env.template

**Reply by amoorefilms** [reply]
where exactly is the .env located? when I go to /opt there is no directory called vaultwarden

**Reply by tteck** [reply]
Which installation method did you use?
Alpine would be `/etc/conf.d/vaultwarden`
Debian would be `/opt/vaultwarden/.env`
